README.md

```diff
@@ -64,109 +69,171 @@ Use the ``Deploy to IBM Cloud`` button **OR** create the services and run locall
+The next step is to configure your Watson Discovery service.
 
-![discovery](doc/source/images/discovery-1.png)
+  1. Find the Discovery service in your IBM Cloud Dashboard.
+  2. Click on the service and then click **Launch tool**.
 
-After the service is created the credentials are available on the *Manage* tab. For more advanced credential management, click on the *Service credentials* tab. These credentials will be needed when configuring the app.
+The Watson Discovery News data collection is already associated with your service. You'll use this collection as the data source for your app. To access the collection, you must find the `COLLECTION_ID` and `ENVIRONMENT_ID`. To find these values:
 
-![discovery-credentials](doc/source/images/discovery-2.png)
+  1. Click on the collection from the Manage Data panel. In this case, it is named `Watson Discovery News`.
+  2. Click on the drop-down icon located in the top right corner of the panel.
 
-## 4. Configure Slack
+![get-collection-id](https://raw.githubusercontent.com/IBM/pattern-utils/master/watson-discovery/get-collection-id.png)
 
-To integrate a new Slack Bot into your existing Slack team, navigate to `https://<my.slack.com>/apps/manage/custom-integrations`, where `<my.slack.com>` is the Slack workspace you want to customize.
+Typically, these values need to be added to the application .env file (as shown in the next step), but in the case of `Discovery News`, the values are always `news-en` (though it might vary based on language) and `system`, so you do not need to specify them.
 
-From the **Cutsom Integrations** page, select the **Bots** option.
+## 4. Add Watson Discovery credentials
 
-![slack-integrations](doc/source/images/slack-0.png)
+Next, you'll need to add the Watson Discovery credentials to the .env file.
 
-To add a new bot, select the **Add Configuration** button.
+  1. From the home directory of your cloned local repo, create a .env file by copying it from the sample version.
 
-![slack-integrations](doc/source/images/slack-00.png)
+<!--remove these HTML tags when series gets support for markdown code blocks -->
+<div class="bx--snippet bx--snippet--multi bx--snippet-btn--expand--hide" data-code-snippet="">
+  <div class="bx--snippet-container" aria-label="Code Snippet Text">
+  <pre><code>
+cp env.sample .env
+  </pre></code>
+  </div>
+</div>
 
-Enter a username for the bot and click **Add bot integration**.
+  2. Locate the service credentials listed on the home page of your Discovery service.
 
-![slack-bot-name](doc/source/images/slack-1.png)
+![get-creds](https://raw.githubusercontent.com/IBM/pattern-utils/master/watson-discovery/get-creds.png)
 
-Once created, save the **API Token** that is generated.
+  3. Copy and paste the `apikey` and `URL` values from your Watson Discovery service credentials into the .env file:
 
-![more-slack-config](doc/source/images/slack-2.png)
+```bash
+# Watson Discovery
+DISCOVERY_URL=&lt;add_discovery_url&gt;
```

Comment:
the ampersand stuff is showing in the markdown render

Reply:
done